projects/RPi2/filesystem/usr/lib/systemd/system/serial-console.service

```diff
@@ -5,7 +5,8 @@ ConditionKernelCommandLine=console
 
 [Service]
 WorkingDirectory=/storage
-ExecStart=/bin/sh -c '. /etc/profile; exec /bin/sh'
+Environment="ENV=/etc/profile"
+ExecStart=/bin/sh -c 'exec /bin/sh'
```

Comment:
Almost correct. /etc/profile is executed. Environment variables are exported and transferred to shell. But if you set some aliases in profile.d folder they are not set in this new shell. That's why I think this PR fixes this shortcoming and calls profile script from actual shell serial console uses.

@haggaie, because this is originally your commit can you comment?